common/core/tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "target":"es5",
         "sourceMap": true,
         "declaration": true,
-        "noUnusedLocals": false,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
```

Comment:
For what it's worth, I've verified that these default to false.

Reply:
Me too.